docs/src/samples/k9s/plugins.yml

```
@@ -68,7 +69,17 @@ plugins:
     background: false
     args:
       - -c
-      - "kubectl cnpg logs cluster $NAME -f -n $NAMESPACE --context $CONTEXT"
+      - "kubectl cnpg logs cluster $NAME -f -n $NAMESPACE --context $CONTEXT"      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)